src/components/backend-ai-session-view.ts

```diff
@@ -444,7 +444,7 @@ export default class BackendAiSessionView extends BackendAIPage {
     return html`
       <lablup-activity-panel title="${_t('summary.ResourceStatistics')}" elevation="1" autowidth>
         <div slot="message">
-          <backend-ai-resource-monitor location="session" id="resource-monitor" ?active="${this.active === true}"></backend-ai-resource-monitor>
+          <backend-ai-resource-monitor location="session" id="resource-monitor" ?active="${this.active === true}" direction="horizontal"></backend-ai-resource-monitor>
```

Comment:
I think that it's not necessary to cast to horizontal. Because you modified it to show the scaling group select box regardless of direction.

Reply:
thank you. I removed the direction cast part. 😁